.pre-commit-config.yaml

```diff
@@ -114,6 +114,11 @@ repos:
           - --license-filepath
           - .github/workflows/license-templates/LICENSE.txt
           - --fuzzy-match-generates-todo
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.19.1
+    hooks:
+      - id: pyupgrade
+        args: [--py37-plus]
```

Comment:
`--py37-plus` will remove Python <= 3.6 only blocks

https://github.com/asottile/pyupgrade?tab=readme-ov-file#python2-and-old-python3x-blocks